README.md

```diff
@@ -13,6 +13,7 @@
 ---
 
 ## Installation and Usage
+Before installing make sure you have at least LTS v14 or greater of Node.js. You can check your node version using ```bash node -v```.
```

Comment:
```suggestion
The minimum supported version of Node.js is v14. You can check your Node.js version with `node --version`.
```

FYI you use single backticks for inline code. And the `bash` part is unnecessary—it can be run in any shell.

Reply:
Thanks, I didn't know you could use single backticks. Those changes make more sense for the section.